client/client_test.go

```diff
@@ -29,3 +31,15 @@ func TestNewClient(t *testing.T) {
 		assert.Check(t, err, c.doc)
 	}
 }
+
+var runOnRealCluster = flag.Bool("use-real-cluster", false, "client package tests will use KUBECONFIG configured cluster")
```

Comment:
As the real is implicit how about a slight mod:

clusterRun flag.Bool("use-cluster", false, "run tests will against a configured cluster")

Also, to ensure flag evaluation, TestMain should be introduced. Additional setup and teardown can also be place here if ever needed:

func TestMain(m *testing.M) {
    flag.Parse()
    // setup here if needed
    os.Exit(m.Run())   
    // teardown 
}

Reply:
Agree, done.